source/Calamari.Common/Features/StructuredVariables/StructuredConfigVariablesService.cs

```diff
@@ -3,6 +3,8 @@
 using System.IO;
 using System.Linq;
 using Autofac;
+using Autofac;
```

Comment:
You appear to have a duplicated reference here which we could remove.

Reply:
Nice, thankyou!

Reply:
Fixed